exporter/azuremonitorexporter/log_to_envelope.go

```diff
@@ -43,7 +43,11 @@ type logPacker struct {
 func (packer *logPacker) LogRecordToEnvelope(logRecord plog.LogRecord) *contracts.Envelope {
 	envelope := contracts.NewEnvelope()
 	envelope.Tags = make(map[string]string)
-	envelope.Time = toTime(logRecord.Timestamp()).Format(time.RFC3339Nano)
+	timestamp := logRecord.Timestamp()
```

Comment:
I remember a conversation about adding an `IsZero()` function. Was it for `pdata`?

Reply:
No consensus was reached to add `IsZero`, but it was clarified that the pdata representation of time is uint64, so checking with `== 0` is correct.